templates/lxc-local.in

```diff
@@ -260,6 +260,35 @@ process_fstab() {
   fi
 }
 
+process_templates() {
+  # Look for extra templates
+  template="$(relevant_file template)"
+  if [ -e "${template}" ]; then
+    while read -r line; do
+      fullpath="${LXC_ROOTFS}/${line}"
+      [ ! -e "${fullpath}" ] && continue
+      TEMPLATE_FILES="${TEMPLATE_FILES};${fullpath}"
+    done < "${template}"
+  fi
+}
+
+unpack_metadata() {
+  # Unpack file that contains the container metadata
+  # If the file does not exist, just warn and continue.
+  if [ -n "${LXC_METADATA}" ] && [ -f "${LXC_METADATA}" ]; then
+    if tar Jxf "${LXC_METADATA}" -C "${LOCAL_TEMP}"; then
+      echo "Unpacked metadata file: ${LXC_METADATA}"
+      process_excludes
+      process_config
+      process_fstab
+      process_templates
+    else
+      echo "Unable to unpack metadata file: ${LXC_METADATA}" 2>&1
+      exit 1
```

Comment:
Same comment as the other command, if you flip the if statement around, and just call `exit 1` in the condition, then you can avoiding needing an else and save a level of indentation for most the function.